src/i18n/backend/yaml.cr

```diff
@@ -10,7 +10,7 @@ module I18n
       getter available_locales
       property translations
 
-      @translations = Hash(String, Hash(String, YAML::Type)).new
+      @translations = Hash(String, Hash(String, YAML::Any)).new
```

Comment:
I think `YAML::Any` should be replaced with `YAML::Any::Type`

Reply:
Well, no. `YAML.parse` returns an `YAML::Any`.